### PR TITLE
Add IPython-7.25.0-GCCcore-10.3.0.eb to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Trying to add IPython-7.25.0-GCCcore-10.3.0.eb to NESSI/2022.11

### CODE REVIEW

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -424,7 +424,7 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
 #$EB Perl-5.32.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2640
 #use enhanced CMake easyblock to patch CMake's UnixPaths.cmake script if --sysroot is set
 #from https://github.com/easybuilders/easybuild-easyblocks/pull/2248
-$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:427]: Let's keep CMake uncommented.

File: eessi-2022.11.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 easyconfigs:
   - EasyBuild-4.7.0.eb
-  - CMake-3.20.1-GCCcore-10.3.0.eb
```
Comment (trz42) [inline eessi-2022.11.yml:3]: Let's keep it in the easystack file.

**Review by trz42 — CHANGES_REQUESTED** [review]
Minor comments.

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh]: Followed the guidelines

File: eessi-2022.11.yml (hunk 2 above)
Comment (TopRichard) [reply eessi-2022.11.yml:3]: Followed the guidelines

**Comment by trz42** [comment]
- [x] did the builds for all required architectures succeed (SUCCESS message + reasonably sized tarball)
  - succeeded for `x86_64/generic`, `x86_64/amd/zen2`, `x86_64/intel/{broadwell,skylake_avx512,cascadelake}`
- [x] is the PR up-to-date with the target branch `nessi.no-2022.11-dev` in the repository (if not what are the differences)
- [ ] are all requested changes sound (checking files changed on GitHub.com)
  - it seems something from a merge conflict has leaked into `EESSI-pilot-install-software.sh`
- [x] is/are the easyconfig/s being built included with the EB version used (if not why not)
  - included with EB v4.7.0
- [x] were any changes needed to get the build(s) succeed (common changes for all architectures, changes for a single architecture, changes because of build environment specifics, etc.)
  - no changes were needed

File: eessi-2022.11.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 easyconfigs:
   - EasyBuild-4.7.0.eb
   - CMake-3.20.1-GCCcore-10.3.0.eb
+  - IPython-7.25.0-GCCcore-10.3.0.eb
   - Python-3.9.5-GCCcore-10.3.0.eb
```
Comment (trz42) [inline eessi-2022.11.yml:5]: Should the order be

```ini
  - Python-3.9.5-GCCcore-10.3.0.eb
  - IPython-7.25.0-GCCcore-10.3.0.eb
```

Seems more logical to build Python first, then IPython?

**Review by trz42 — CHANGES_REQUESTED** [review]
Minor tweak in easystack file needed.

**Comment by trz42** [comment]
- [x] did the builds for all required architectures succeed (SUCCESS message + reasonably sized tarball)
- [x] is the PR up-to-date with the target branch `nessi.no-2022.11-dev` in the repository (if not what are the differences)
- [x] are all requested changes sound (checking files changed on GitHub.com)
- [x] is/are the easyconfig/s being built included with the EB version used (if not why not)
- [x] were any changes needed to get the build(s) succeed (common changes for all architectures, changes for a single architecture, changes because of build environment specifics, etc.)

@TopRichard nearly done! The order of `eb` files in `EESSI-pilot-install-software.sh` and `eessi-2022.11.yml` should be consistent. That's an easy fix, no need to rebuild. Other than that, the software is ready to get deployed.

**Pull request closed without merging**